src/components/header/SelectionScrubberOverlay.js

```diff
@@ -2,14 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
 import classNames from 'classnames';
 import clamp from 'clamp';
 import Draggable from '../shared/Draggable';
 import { getFormattedTimeLength } from '../../profile-logic/range-filters';
+import { updateProfileSelection } from '../../actions/profile-view';
+import type { Milliseconds } from '../../types/units';
+
+type Props = {
+  rangeStart: number,
+  rangeEnd: number,
+  selectionStart: number,
+  selectionEnd: number,
+  isModifying: boolean,
+  width: number,
+  onSelectionChange: typeof updateProfileSelection,
+  onZoomButtonClick: (start: Milliseconds, end: Milliseconds) => void,
```

Comment:
I discussed with greg recently, and we agreed it's usually better to let Flow infer the return value using `*` instead of `void` when we don't use the return value.